tutorials/getting-started-with-c7n.md

```diff
@@ -0,0 +1,289 @@
+
+1. Tag one with the key `c7n-tutorial` and leave the value empty
+2. Tag one with the key `c7n-tutorial` and with the value `my-first-policy-pull` 
+
+## Anatomy of a Cloud Custodian Policy
+
+As mentioned and illustrated above, Cloud Custodian synthesizes the many ad-hoc scripts you would need to write to orchestrate API calls to a cloud provider in order to govern your infrastructure into a domain specific language. This language uses a declarative vocabulary of `resource`, `filters`, and `actions` to configure policies in YAML. This vocabulary is explained more below:
+
+YAML Key | Description
+--- | ---
+resource | A cloud resource or service. In a Cloud Custodian policy file, this key refers to the type of cloud resource or service the following `actions` and `filters` values will act upon. Run `custodian schema` to reveal a list of resources available to Cloud Custodian.
+filters | Attributes of a resource that can be used as criteria to filter resources by. The result of a filter is an output of resources that meet the filter criteria. This output is what `actions` will act upon. At their core, `filters` are key value pairs that can be used more generically as well. See the [Cloud Custodian documentation](https://cloudcustodian.io/docs/filters.html) for more information. Run `custodian schema ec2.filters` to reveal a list of filters available for EC2 instances.
+actions | Operations that can be performed upon a resource. These operations will be performed on the output of `filters`. At their core, `actions` are webhooks that can be used more generically as well. See the [Cloud Custodian documentation](https://cloudcustodian.io/docs/actions.html) for more information. Run `custodian schema ec2.actions` to reveal a list of actions available for EC2 instances.
+
+## Your First Cloud Custodian Policy
+
+As mentioned above, for your first policy, you will filter for EC2 instances with a specific tag key and value. When and if the policy finds any such instances, the policy will stop the instance(s) and update the tag key with a new value.
+
+Now that you have been exposed to the anatomy of a Cloud Custodian policy, try writing one on your own. Create a file named `my-first-policy-pull.yml` and give it a try.
+
+If you are using AWS CloudShell, the easiest way to write your policy file is to use your text editor of choice and then upload the file to CloudShell via the Actions dropdown menu available in the CloudShell UI.
+
+If your policy looks like the one below, great job!
+
+```
+policies:
+  - name: my-first-policy-pull-stop-tagged-ec2
+    resource: aws.ec2
+    filters:
+      - tag:c7n-tutorial: "my-first-policy-pull"
+    actions:
+      - stop
```

Comment:
You might want to swap these as well so it tags it before it stops it though this might not be an actual issue. Not sure if AWS allows tagging a resource while it is in the "stopping" state